config/config.go

```diff
@@ -281,5 +281,28 @@ func migrateConfigDir() error {
 	// ensure ~/.config exists
 	os.MkdirAll(path.Dir(DefaultConfigDirPath), 0700)
 
-	return os.Rename(LegacyConfigDirPath, DefaultConfigDirPath)
+	err = os.Rename(LegacyConfigDirPath, DefaultConfigDirPath)
+	if err != nil {
+		return err
+	}
+
+	// adapt certificate paths in kubeconfig
+	if len(KubeConfigPaths) > 0 {
+		// we only adapt the first file found (on purpose)
+		theKubeConfigPath := KubeConfigPaths[0]
+		if stat, err := os.Stat(theKubeConfigPath); err == nil {
+			oldConfig, configErr := ioutil.ReadFile(theKubeConfigPath)
+			if configErr != nil {
+				return configErr
+			}
+			newConfig := strings.Replace(string(oldConfig), LegacyConfigDirPath, DefaultConfigDirPath, -1)
+			// write back
+			writeErr := ioutil.WriteFile(theKubeConfigPath, []byte(newConfig), stat.Mode())
+			if writeErr != nil {
+				return fmt.Errorf("Could not overwrite kubectl config file. %s", writeErr.Error())
```

Comment:
Usually go errors doesn't start with capital letter and terminate with full stop.